README.rst

```diff
@@ -79,13 +79,13 @@ After installation, enable the context menu entries from a terminal as follows:
 
 .. code:: bash
 
-    $ jupyter_context-menu_add
+    $ start_jupyter_cm
```

Comment:
To be symmetric with `remove`, add `--add` which would do the same as without argument?